build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml

```diff
@@ -64,6 +64,11 @@ jobs:
         Copy-Item -Path "$targetsFilePath\Microsoft.WindowsAppSDK.Bootstrap.targets" -Destination "$fullpackagePath\build\Microsoft.WindowsAppSDK.Bootstrap.targets"
 
         Copy-Item -Path "$targetsFilePath\AppxManifest.xml" -Destination "$fullpackagePath\AppxManifest.xml"
+        $manifestPath = $fullpackagePath+'\manifests';
+        New-Item -ItemType Directory -Force -Path $manifestPath;
+        $xslt = New-Object System.Xml.Xsl.XslCompiledTransform;
+        $xslt.Load('build\TransformAppxManifest.xslt');
+        $xslt.Transform($fullpackagePath+'\AppxManifest.xml', $manifestPath+'\WindowsAppSDK.manifest');
```

Comment:
I would name the output something like Microsoft.WindowsAppSdk.Foundation.manifest, just to ensure it doesn't collide with other transport packages, and to provide a pattern to follow

Reply:
Good thinking.  Updated

Reply:
@Scottj1s where should that be noted in the Transport Package Spec? ;-)

Reply:
@DrusTheAxe I'm planning to make spec updates once the end-to-end is working